ftw/slider/upgrades/20200527190657_remove_plone_app_referenceablebehavior/types/ftw.slider.Container.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<object name="ftw.slider.Container"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        xmlns:zcml="http://xml.zope.org/namespaces/zcml"
+        i18n:domain="ftw.slider" >
+
+  <!-- Basic metadata -->
+  <property name="title" i18n:translate="">Slider Container</property>
+  <property name="description" i18n:translate=""></property>
+  <property name="content_icon">folder.png</property>
+  <property name="allow_discussion">False</property>
+  <property name="global_allow">True</property>
+  <property name="filter_content_types">True</property>
+  <property name="allowed_content_types">
+    <element value="ftw.slider.Pane" />
+  </property>
+
+  <!-- schema interface -->
+  <property name="schema">ftw.slider.contents.container.IContainerSchema</property>
+
+  <!-- class used for content items -->
+  <property name="klass">ftw.slider.contents.container.Container</property>
+
+  <!-- add permission -->
+  <property name="add_permission">ftw.slider.AddContainer</property>
+
+  <!-- enabled behaviors -->
+  <property name="behaviors">
+    <element value="plone.app.dexterity.behaviors.metadata.IBasic" />
+    <element value="plone.app.content.interfaces.INameFromTitle" />
+    <element value="plone.app.dexterity.behaviors.exclfromnav.IExcludeFromNavigation"/>
+    <element zcml:condition="installed plone.multilingualbehavior"
+             value="plone.multilingualbehavior.interfaces.IDexterityTranslatable" />
+  </property>
+
+  <!-- View information -->
+  <property name="default_view">view</property>
+  <property name="default_view_fallback">False</property>
+  <property name="view_methods">
+    <element value="view"/>
+  </property>
+
+  <!-- Method aliases -->
+  <alias from="(Default)" to="(dynamic view)"/>
+  <alias from="edit" to="@@edit"/>
+  <alias from="sharing" to="@@sharing"/>
+  <alias from="view" to="(selected layout)"/>
+
+  <!-- Actions -->
+  <action
+      action_id="view"
+      title="View"
+      category="object"
+      condition_expr=""
+      url_expr="string:${object_url}"
+      visible="True">
+    <permission value="View"/>
+  </action>
+
+  <action
+      action_id="edit"
+      title="Edit"
+      category="object"
+      condition_expr=""
+      url_expr="string:${object_url}/edit"
+      visible="True">
+    <permission value="Modify portal content"/>
+  </action>
+
+</object>
```

Comment:
You only have to add the delta, otherwise you will overwrite policy level customizations. 
```suggestion
<?xml version="1.0"?>
<object name="ftw.slider.Container">
  <property name="behaviors" purge="False">
    <element value="plone.app.referenceablebehavior.referenceable.IReferenceable" remove="True"/>
  </property>
</object>
```

Reply:
Thanks, changed it to only the diff.